src/main/resources/application.yaml

```diff
@@ -99,6 +99,9 @@ audiotransform:
   re-encode-workspace: ${user.home}/audiotransform/encode
 
 darts:
+  gateway:
+    url: http://localhost:8070
```

Comment:
we would need this in terraform scripts or something, but i guess its fine here for now as we don't have any server names yet?
@davet1985

Reply:
Yes, we need to specify this is chart/darts-api/values.yaml, as environment, something like this

```
  environment:
    DARTS_GATEWAY_URL: https://darts-gateway.{{ .Values.global.environment }}.platform.hmcts.net
```

Then add a tweak here to set the value.

Reply:
Done in commit e3591a1